recipes/variant-lite/all/conanfile.py

```diff
@@ -20,18 +22,22 @@ def package_id(self):
         self.info.header_only()
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = self.name + "-" + self.version
-        os.rename(extracted_dir, self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
 
     def package(self):
         self.copy("*.hpp", dst="include", src=os.path.join(self._source_subfolder, "include"))
         self.copy("LICENSE.txt", dst="licenses", src=self._source_subfolder)
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "variant-lite")
+        self.cpp_info.set_property("cmake_target_name", "nonstd::variant-lite")
+
+        # TODO: to remove in conan v2 once cmake_find_package* generators removed
```

Comment:
👋 If I followed the changes correctly...

The 7 lines below will be replaced with just two?! 🤯

Reply:
yes